src/edu.umn.cs.melt.copper.compiletime/src/main/java/edu/umn/cs/melt/copper/main/ParserCompiler.java

```diff
@@ -78,8 +78,9 @@ private static String usageMessage(ParserCompilerParameters args)
 		rv += "\t-mda\tRun Copper's modular determinism analysis on the input.\n\t\t\tIf this switch is used, the input must comprise exactly\n\t\t\ttwo grammars: the host and an extension to test.\n";
 		rv += "\t-avoidRecompile\tRun Copper only if one spec-file has a later\n\t\t\tmodification time than the output file.\n\t\t\tIf this switch is used, an output file must also\n\t\t\tbe specified.\n";
 		rv += "\t-logfile [lout]\tPipe all log output to the file 'lout'\n\t\t\t(default standard error).\n";
-		rv += "\t-dot [dout]\tOutput Graphiz dot code representing any counterexamples to 'dout'\n\t\t\t(no dot output if not provided).\n";
-		rv += "\t-nc\t\tDo not use ascii color escape sequences to color any counterexamples\n";
+		rv += "\t-dot [dout]\tOutput dot markup code representing any counterexamples to 'dout'\n\t\t\t(no dot output if not provided).\n";
+		rv += "\t-nc \t\tDo ot use ANSI color escape sequences to color any counterexamples\n\t\t\t(color is enabled by default for standard out, by default if using a logfile).\n";
```

Comment:
Should be "disabled by default if using a logfile"?

Reply:
As the default log output is *stderr*, not stdout (and as the question of log colorization is not necessarily limited to the counterexamples) I would use the following text for the `-cc` and `-nc` options, which should make clear what the defaults are:
```
-cc Use ANSI color escape sequences in log output, even when logging to a file
-nc Suppress ANSI color in log output, even when logging to the terminal
```

Also, on commit 52ffd6ea, the color test seems to be the wrong way around -- I tried it out with a conflicting grammar, and the counterexamples were without color on the terminal unless I used `-cc` and with color in the logfile unless I used `-nc`.

Reply:
I like that wording better, changed. 
Yep, looks like i put a `==` where i needed a `!=`, that also should be fixed now.